Samples/ReVerC/Operation.qs

```diff
@@ -0,0 +1,44 @@
+﻿namespace Quantum.ReVerC
+{
+    open Microsoft.Quantum.Primitive;
+    open Microsoft.Quantum.Canon;
+
+  /// Prepares a quantum register in the given basis state
+  operation prepare(initial: Result[], register: Qubit[]) : ()
```

Comment:
Q# naming conventions follow general .NET conventions, in that operation and function names follow `CamelCase` capitalization. Would you be willing to change the names of `prepare` and `add` to `Prepare` and `Add` accordingly?